arm-servicefabric/2016-09-01/swagger/servicefabric.json

```diff
@@ -404,43 +404,43 @@
       }
     },
     "EndpointRangeDescription": {
-      "description": "Endpoint range description",
+      "description": "Port range details",
       "properties": {
         "startPort": {
-          "description": "Endpoint start port",
+          "description": "Starting port of a range of ports",
           "type": "integer"
         },
         "endPort": {
-          "description": "Endpoint end port",
+          "description": "End port of a range of ports",
           "type": "integer"
         }
       }
     },
     "NodeTypeDescription": {
-      "description": "NodeType description",
+      "description": "NodeType details",
       "properties": {
         "name": {
-          "description": "NodeType description name",
+          "description": "Name of NodeType",
           "type": "string"
         },
         "placementProperties": {
-          "description": "Placement properties key",
+          "description": "Placement property key",
```

Comment:
`placementProperties` will become member property of the model `NodeTypeDescription` like

```C#
class NodeTypeDescription {
  // Placement property key
  //
  Map<String, String> placementProperties;
}
```

The description for `placementProperties` is wrong here, it says this property is a key. 

Can we have a better description? e.g. "The placement tags applied to nodes in the node type, which can be used to indicate where certain services (workload) should run"